README.md

```diff
@@ -14,10 +14,15 @@ Postgres 9.5+
 ### 3. Create database, migrate tables and seed user data
 `rake db:setup`
 
-### 4. Run the application
+### 4. Set Rails master key environment variable
+```
+export RAILS_MASTER_KEY=`registers-pass show registers/app/manager/store`
+```
```

Comment:
Based on my experience from yesterday I think this step must come before `rake db:setup`. Otherwise it looks great 👍

Reply:
@arnau I have made this change now, can you review again?